stm32-modules/include/heater-shaker/heater-shaker/host_comms_task.hpp

```diff
@@ -138,6 +138,7 @@ class HostCommsTask {
     }
 
     [[nodiscard]] auto may_connect() const -> bool { return may_connect_latch; }
+    [[nodiscard]] auto uart_may_connect() const -> bool { return uart_may_connect_latch; }
```

Comment:
Is there a scenario where `uart_may_connect_latch` won't be the same as `may_connect_latch`? Might be able to omit this.